app/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM python:3.10-slim
 WORKDIR /app
 ENV PATH "$PATH:/app"
 COPY requirements.txt requirements.txt
+RUN echo "deb http://deb.debian.org/debian bookworm main" > /etc/apt/sources.list
```

Comment:
why this change is needed ?

Reply:
The build is failing actually. It is not able to find /etc/apt/sources.list. This is not final yet, checking why build is failing

Reply:
reverted these changes